dotnet/src/Connectors/Connectors.AI.OpenAI/AzureSdk/OpenAIFunctionResponse.cs

```diff
@@ -27,6 +28,14 @@ public class OpenAIFunctionResponse
     /// </summary>
     public Dictionary<string, object> Parameters { get; set; } = new();
 
+    /// <summary>
+    /// Fully qualified name of the function. This is the concatenation of the plugin name and the function name,
+    /// separated by the value of <see cref="OpenAIFunction.NameSeparator"/>.
+    /// If there is no plugin name, this is the same as the function name.
+    /// </summary>
+    public string FullyQualifiedName =>
+        this.PluginName.IsNullOrEmpty() ? this.FunctionName : string.Join(OpenAIFunction.NameSeparator, this.PluginName, this.FunctionName);
```

Comment:
`string.Join(..., params string[])` is unnecessary expense here.  This can instead be:
```suggestion
        this.PluginName.IsNullOrEmpty() ? this.FunctionName : $"{this.PluginName}{OpenAIFunction.NameSeparator}{this.FunctionName}";
```
or
```suggestion
        this.PluginName.IsNullOrEmpty() ? this.FunctionName : this.PluginName + OpenAIFunction.NameSeparator + this.FunctionName;
```
or
```suggestion
        this.PluginName.IsNullOrEmpty() ? this.FunctionName : string.Concat(this.PluginName, OpenAIFunction.NameSeparator, this.FunctionName);
```
all of which will compile down identically.